Gemini/cfg/cfg.php

```diff
@@ -9,4 +9,25 @@
         'user' => 'changeme',
         'password' => 'changeme',
     ],
+
+    // Valid log levels:
+    //  - DEBUG
+    //  - INFO
+    //  - NOTICE
+    //  - WARNING
+    //  - ERROR
+    //  - CRITICAL
+    //  - ALERT
+    //  - EMERGENCY
+    //  - NONE
+    // if none is used the log file won't be opened.
+    'loglevel' => 'NONE',
+    'logfile' => '../gemini.log',
+
+    // Toggles JWT security for the service.
+    'security enabled' => true,
+    // Path to the syn config file for authentication.
+    // Example can be found here:
+    // https://github.com/Islandora-CLAW/Syn/blob/master/conf/syn-settings.example.xml
+    'security config' => '../syn-settings.xml',
```

Comment:
Is there supposed to be a copy of this provided by default? Without it you get a 500, so if we aren't providing a sample one, we should catch it more cleanly.

```
ubuntu@claw:~$ curl -i -XPOST -H"Content-type: application/json" -d '{ "drupal" : "test/resource", "fedora" : "preservation/test/resource" }' http://localhost:8083HTTP/1.0 500 Internal Server Error
Host: localhost:8083
Connection: close
X-Powered-By: PHP/7.0.15-0ubuntu0.16.04.4
Content-type: text/html; charset=UTF-8
```
On the PHP server tab
```
[Fri Apr 14 20:19:27 2017] PHP Warning:  file_get_contents(../syn-settings.xml): failed to open stream: No such file or directory in /var/www/html/Crayfish/Gemini/vendor/islandora/crayfish-commons/src/IslandoraServiceProvider.php on line 60
[Fri Apr 14 20:19:27 2017] PHP Fatal error:  Uncaught Error: Call to a member function getName() on boolean in /var/www/html/Crayfish/Gemini/vendor/islandora/crayfish-commons/src/Syn/SettingsParser.php:48
Stack trace:
#0 /var/www/html/Crayfish/Gemini/vendor/islandora/crayfish-commons/src/IslandoraServiceProvider.php(63): Islandora\Crayfish\Commons\Syn\SettingsParser->__construct(false, Object(Monolog\Logger))
#1 /var/www/html/Crayfish/Gemini/vendor/pimple/pimple/src/Pimple/Container.php(113): Islandora\Crayfish\Commons\IslandoraServiceProvider->Islandora\Crayfish\Commons\{closure}(Object(Silex\Application))
#2 /var/www/html/Crayfish/Gemini/vendor/islandora/crayfish-commons/src/IslandoraServiceProvider.php(69): Pimple\Container->offsetGet('crayfish.syn.se...')
#3 /var/www/html/Crayfish/Gemini/vendor/pimple/pimple/src/Pimple/Container.php(113): Islandora\Crayfish\Commons\IslandoraServiceProvider->Islandora\Crayfish\Commons\{closure}(Object(Silex\Application))
#4 /var/www/html/Crayfish/Gemini/vendor/silex/silex/src/Silex/Provider/SecurityServiceProvider.ph in /var/www/html/Crayfish/Gemini/vendor/islandora/crayfish-commons/src/Syn/SettingsParser.php on line 48
```